docs/developer/tutorials/snmp/profile-format.md

```diff
@@ -470,6 +470,48 @@ This example will submit two metrics `snmp.upsBasicStateOutputState.OnLine` and
 
 [Example of flag_stream usage in a profile](https://github.com/DataDog/integrations-core/blob/e64e2d18529c6c106f02435c5fdf2621667c16ad/snmp/datadog_checks/snmp/data/profiles/apc_ups.yaml#L60-L127).
 
+#### Extract value
+
+If the metric value to be submitted is from a OID with string value and needs to be extracted from it, you can use extract value feature.
+
+`extract_value` is a regex pattern with one capture group like `(\d+)C`, where the capture group is `(\d+)`.
+
+Example use cases respective regex patterns:
+
+- stripping the C unit from a temperature value: `(\d+)C`
+- stripping the USD unit from a currency value: `USD(\d+)`
+- stripping the F unit from a temperature value with spaces between the metric and the unit: `(\d+) *F`
+
+Example:
+
+**Scalar Metric Example:**
+
+```yaml
+metrics:
+  - MIB: MY-MIB
+    symbol:
+      OID: 1.2.3.4.5.6.7
+      name: temperature
+      extract_value: '(\d+)C'
+```
+
+**Table Column Metric Example:**
+
+```yaml
+metrics:
+  - MIB: MY-MIB
+    table:
+      OID: 1.2.3.4.5.6
+      name: myTable
+    symbols:
+      - OID: 1.2.3.4.5.6.7
+        name: temperature
+        extract_value: '(\d+)C'
```

Comment:
Are simple quotes necessary?

Reply:
@gmarabout The single quotes are not necessary, but IMHO, it makes it a bit more explicit for strings containing some special characters.